data/result-exporter/swe-cne-exporter/src/main/java/com/farao_community/farao/data/swe_cne_exporter/SweCneHelper.java

```diff
@@ -29,9 +30,9 @@ public class SweCneHelper extends CneHelper {
     private final AngleMonitoringResult angleMonitoringResult;
     private Map<Contingency, Boolean> contingencyFailureMap = new HashMap<>();
 
-    public SweCneHelper(Crac crac, Network network, RaoResult raoResult, AngleMonitoringResult angleMonitoringResult, RaoParameters raoParameters, CneExporterParameters exporterParameters) {
+    public SweCneHelper(Crac crac, Network network, RaoResult raoResult, RaoParameters raoParameters, CneExporterParameters exporterParameters) {
         super(crac, network, raoResult, raoParameters, exporterParameters);
-        this.angleMonitoringResult = angleMonitoringResult;
+        this.angleMonitoringResult = ((RaoResultWithAngleMonitoring) raoResult).getAngleMonitoringResult();
```

Comment:
you shouldn't need angleMonitoringResult anymore, angle values can be accessed in RaoResult